### PR TITLE
Server certificate

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Implemented Server Certificate Resource for API 600/800/1000/1200

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [y ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [y] New endpoints supported are updated in the endpoints-support.md file.
- [y] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api600/c7000/server_certificate.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        def import
+          @data.delete("aliasName")
+          create(header = self.class::DEFAULT_REQUEST_HEADER)
+          true
+        end
+        def delete(*)
+          unavailable_method
+        end
+        def retrieve!
+          response = @client.rest_get(self.class::BASE_URI + "/#{@data["aliasName"]}")
```
Comment (soodpr) [inline lib/oneview-sdk/resource/api600/c7000/server_certificate.rb]: raise a ArgumentError if aliasname is not passed

File: endpoints-support.md [hunk 2]
```diff
@@ -325,6 +325,14 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | PUT      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/scopes/resources/{resource_uri}</sub>                                         | PATCH    | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:| :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+
+|     **Server Certificate**                                                                                                   |
```
Comment (AnushaY1916) [inline endpoints-support.md:329]: There are formatting errors here. Please correct it.

Comment (AsisBagga) [reply endpoints-support.md:328]: @AnushaY1916 Corrected.

File: examples/shared_samples/server_certificate.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+# specific language governing permissions and limitations under the License.
+
+require_relative '../_client' # Gives access to @client
+
+# NOTE: You'll need to add the following instance variable to the _client.rb file with valid values for your environment:
+#   @storage_system_ip
+
+# All supported APIs for  Server Certificate:
+# - 600, 800, 1000, 1200
+# # Resources classes that you can use for Client Certificate in this example:
```
Comment (soodpr) [inline examples/shared_samples/server_certificate.rb:19]: its server certificate. please correct it everywhere

File: CHANGELOG.md [hunk 4]
```diff
@@ -23,6 +26,7 @@ This release does not support any new features added in API versions 800,1000 an
 - OS Volume
 - Plan Script
 
+
```
Comment (soodpr) [inline CHANGELOG.md:29]: remove this extra line

File: examples/_client.rb.example [hunk 5]
```diff
@@ -4,7 +4,7 @@ require 'pry'
 @client = OneviewSDK::Client.new(
   url: 'https://oneview.example.com', # or set ENV['ONEVIEWSDK_URL']
   user: 'Administrator',  # or set ENV['ONEVIEWSDK_USER']
-  password: 'secret123',  # or set ENV['ONEVIEWSDK_PASSWORD']
+  password: 'secret@123',  # or set ENV['ONEVIEWSDK_PASSWORD']
```
Comment (soodpr) [inline examples/_client.rb.example:7]: avoid doing such modifications which are unnecessary.

File: examples/shared_samples/server_certificate.rb (hunk 3 above)
Comment (AsisBagga) [reply examples/shared_samples/server_certificate.rb]: @soodpr Corrected

File: CHANGELOG.md (hunk 4 above)
Comment (AsisBagga) [reply CHANGELOG.md]: @soodpr Corrected

File: examples/_client.rb.example (hunk 5 above)
Comment (AsisBagga) [reply examples/_client.rb.example]: @soodpr Corrected

**Pull request merged**